packages/custom-elements-manifest-tools/src/index.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
```

Comment:
```suggestion
 * Copyright 2022 Google LLC
```

Reply:
The copyright years are from when I originally wrote the code and pushed them to the wc-catalog repo. I thought we were supposed to keep the original years.

Reply:
Ah, in that case ok. I didn't realize had already been published.